### ISSUE TITLE
TableauRestApi.signout() fails with: AttributeError: 'bool' object has no attribute 'encode'

### ISSUE THREAD

**Issue by dotariel** [bryantbhowell/tableau_rest_api#10]
An AttributeError exception is thrown when calling `TableauRestApi.signout()` 

Partial traceback:

```
  File "/usr/local/lib/python2.7/site-packages/tableau_rest_api/tableau_rest_api.py", line 702, in signout
    api.request_from_api()
  File "/usr/local/lib/python2.7/site-packages/tableau_rest_api/tableau_rest_api.py", line 2931, in request_from_api
    self.__make_request(page_number)
  File "/usr/local/lib/python2.7/site-packages/tableau_rest_api/tableau_rest_api.py", line 2853, in __make_request
    request.add_header('X-tableau-auth', self.__token.encode('utf8'))
```

It appears that the following line is expecting `self.__token` to be a String, but is being set to a Boolean somewhere:

tableau_rest_api.py:2853
```
if self.__token is not None:
    request.add_header('X-tableau-auth', self.__token.encode('utf8'))
```

**Comment by bryantbhowell** [comment]
I hadn't looked at the signout code in a long time (it's not particularly necessary in most cases). Fixed now in version 2.1.4.

**Comment by dotariel** [comment]
Confirmed. Thanks!
